lintro/ai/review/github.py

```
@@ -0,0 +1,184 @@
+        )
+        for answer in result.checklist:
+            evidence = answer.evidence.replace("|", "\\|")
+            lines.append(f"| {answer.id} | {answer.answer} | {evidence} |")
+
+    outside_diff = [
+        finding for finding in result.findings if not finding.file or finding.line <= 0
+    ]
+    if outside_diff:
+        lines.extend(["", "### Findings outside diff"])
+        for finding in outside_diff:
+            lines.append(f"- **{finding.severity}** {finding.title} ({finding.file})")
```

Ruff (0.15.20)

[warning] Use `list.extend` to create a transformed list

(PERF401)